### ISSUE TITLE
Add blurhash support

### ISSUE THREAD

**Issue by lukepighetti** [lukepighetti/mastodon-flutter#10]
Mastodon has blurhash support, we may be able to use that as a way to handle sensitive content alongside it's intended purpose to provide previews before images have loaded

https://github.com/lukepighetti/mastodon_dart/issues/38
https://pub.dev/packages/flutter_blurhash
https://github.com/lukepighetti/mastodon_flutter/blob/master/lib/src/media.dart#L33-L53

**Comment by GroovinChip** [comment]
Yes, I've actually already added blurhash support to the attachment class in mastodon_dart here: https://github.com/lukepighetti/mastodon_dart/blob/82669da63e378e2c9a2d47cb36b12ba1ff038df4/lib/src/data/attachment.dart#L51

See https://docs.joinmastodon.org/entities/attachment/ for source

**Comment by GroovinChip** [comment]
I don't think I've pushed up the branch that has my work on blurring sensitive images - want me to do that for you @lukepighetti? You could probably improve it a lot.
